### PR TITLE
milestone2

### PR DESCRIPTION
**Pull request by 1v4n4**
In this milestone, I created components , containers, actions, reducer, and store. The app can display the default set of books. It is deployed on Heroku (link in readme).

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@1v4n4 well done on this milestone and I commend your effort in trying to met the requirements, however, few changes are required
### highlight 
Git flow 👍🏻 
project structure 👍🏻 

### To Improve
#### src/index.js
kindly changed `bookstore.js` to `App.js` as recommended by the project requirement
### bookstore` component
 kindly change `bookstore.js` component to `App.js` component

### BooksList component
* [ ] it connects to Redux store, using the `connect()` function or [Redux hooks](https://react-redux.js.org/api/hooks), and fetches books from the state [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToReduxHooks.png) from [pathwright](https://microverse.pathwright.com/library/react-redux-v10/188148/path/step/107676536/)
* [ ] it uses `mapStateToProps` to fetch books from the `store`
* [ ] kindly remove `console.log  from your code

`index.js` reducer 

- it combines app specific reducers, uses `combineReducer`and kindly remove `console.log` in your file
kindly find my inline comments

#### once specific changes are made request for another review

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-import App from './App';
+import { Provider } from 'react-redux';
+import Bookstore from './components/Bookstore';
+import store from './reducers/index';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <Bookstore />
```
Comment (taiwo2) [inline src/index.js]: change the component file name to App.js

File: src/containers/BookList.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from '../components/Book';
+import store from '../reducers';
+
+const BookList = () => {
+  const list = store.getState();
+  console.log(list);
```
Comment (taiwo2) [inline src/containers/BookList.js]: remove console.log

File: src/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import BookReducer from './book';
+
+const store = createStore(BookReducer);
+console.log('store state from reducers', store.getState());
```
Comment (taiwo2) [inline src/reducers/index.js]: remove the console and use  `combinedReducers` to  combines app specific reducers

**Review by taiwo2 — CHANGES_REQUESTED** [review]

File: src/containers/BookList.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from '../components/Book';
+import store from '../reducers';
+
+const BookList = () => {
+  const list = store.getState();
```
Comment (taiwo2) [inline src/containers/BookList.js]: its connect redux to store using `connect()` function  and it uses `mapStateToProps` to fetch books from the `store`

**Review by taiwo2 — CHANGES_REQUESTED** [review]

**Review by bellom — APPROVED** [review]
## Hi @1v4n4  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best**

**Pull request merged**